packages/okta-react/README.md

```diff
@@ -55,26 +63,49 @@ npm install --save @okta/okta-react
 
 ## Usage
 
-`okta-react` works directly with [react-router][] and provides some additional components:
+`okta-react` provides the means to connect a React SPA with Okta OIDC information.  Most commonly, you will connect to a router library such as [react-router][].
+
+### React-Router components (optional)
+
+`okta-react` provides a number of pre-built components to connect a `react-router`-based SPA to Okta OIDC information.  You can use these components directly, or use them as a basis for building your own components.
+- [SecureRoute](#SecureRoute) - A normal `Route` except authentication is needed to render the component.  
+
+### General components
 
-- [Security](#security) - Allows you to supply your OpenID Connect client [configuration](#reference). Provides an instance of the [Auth service][] to child components.
-- [SecureRoute](#secureroute) - A normal `Route` except authentication is needed to render the component.
-- [ImplicitCallback](#implicitcallback) - A simple component which handles the login callback.
+`okta-react` provides the necessary tools to build an integration with most common React-based SPA routers.
 
-## Create Routes
+- [Security](#security) - Allows you to supply your OpenID Connect client [configuration](#reference). Includes React context providers to allow the use of the [useOktaAuth][] React Hook, or the [withOktaAuth][] Higher Order Component wrapper.
+- [LoginCallback](#LoginCallback) - A simple component which handles the login callback when the user is redirected back to the application from the Okta login site.
 
-Here is a minimal working example. This example defines 3 routes:
+Users of routers other than `react-router` will have to add their own checks for authentication.  The exact method will depend on your router, but the basic idea is to use [useOktaAuth][] to see if a `authState.isPending` is false and `authState.isAuthenticated` is true.  If both are false, you can send them to login via `authService.login(...)`.
```

Comment:
Instead of "will have to add...", how about something like:
"can use `authState` to check authentication before loading a route"